pkg/kubectl/describe.go

```diff
@@ -508,7 +507,7 @@ func describePod(pod *api.Pod, events *api.EventList) (string, error) {
 			}
 		}
 		describeVolumes(pod.Spec.Volumes, out, "")
-		fmt.Fprintf(out, "QoS Class:\t%s\n", qos.InternalGetPodQOS(pod))
+		fmt.Fprintf(out, "QoS Class:\t%s\n", pod.Status.QOSClass)
```

Comment:
is this field settable only via the kubelet?  do i only get a value after the pod has been scheduled?